lib/airplay/browser.rb

```diff
@@ -128,10 +128,16 @@ def create_device(name, address, type)
     # Returns nothing
     #
     def resolve(node)
-      resolver = DNSSD::Service.new
-      resolver.resolve(node) do |resolved|
-        break unless node_resolver(node, resolved)
+      begin
+        resolver = DNSSD::Service.new
+        resolver.resolve(node) do |resolved|
+          break unless node_resolver(node, resolved)
+        end
       end
+    rescue ArgumentError
+      # Ignore error
+    rescue DNSSD::UnknownError
+      # Ignore this too
```

Comment:
could/should we log these somewhere?

Reply:
Possibly, though remember this is not technically inside media server itself so we would have to be careful as to how we did it. Also, if we would simply stuff the occurance in a local log fie, I'd say the effort would be wasted. we would want to have them logged through into nagios somehow.

Reply:
thinking further on this, really the things we really want to report on are those errrors that we DON'T catch. that cause the actor to abend. THOSE are the things we want reporting on.